app-foundation/3-networks/development/boa_networking.tf

```diff
@@ -0,0 +1,330 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  boa_gke_cluster1_master_cidr = "100.64.78.0/28"
+  boa_gke_cluster2_master_cidr = "100.65.70.0/28"
+  boa_gke_mci_master_cidr      = "100.64.70.0/28"
+}
+
+/******************************************
+ VPC firewall rules
+*****************************************/
+
+resource "google_compute_firewall" "allow_asm_healthcheck_sidecar_east" {
+  name      = "fw-${var.environment_code}-shared-base-allow-asm-healthcheck-autosidecar-east"
+  project   = local.base_project_id
+  network   = module.base_shared_vpc.network_self_link
+  priority  = 1000
+  direction = "INGRESS"
+
+  dynamic "log_config" {
+    for_each = var.firewall_enable_logging == true ? [{
+      metadata = "INCLUDE_ALL_METADATA"
+    }] : []
+
+    content {
+      metadata = log_config.value.metadata
+    }
+  }
+
+  source_ranges = ["172.16.2.0/28"]
+  target_tags   = ["gke-gke-boa-us-east1-001-1d5c9eff-node"]
```

Comment:
these are autogenerated GKE tags for nodes and should be replaced with custom tags, here and throughout

Reply:
@bharathkkb I updated these to `"gke-boa-us-east1-node"` and `"gke-boa-us-west1-node"` - they will also have to be updated within the module that creates the networking tags for the subnets.